test/functional/lua/vim_spec.lua

```diff
@@ -2769,6 +2769,19 @@ describe('lua stdlib', function()
         eq({'notification', 'wait', {-2}}, next_msg(500))
       end)
     end)
+
+    it('should not run the main loop in fast callbacks (#26122)', function()
```

Comment:
`#` is enough to disambiguate
```suggestion
    it('should not run the main loop in fast callbacks #26122', function()
```